<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PaintStratigraphicOrderWidget</class>
 <widget class="QWidget" name="PaintStratigraphicOrderWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Paint Stratigraphic Order</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="descriptionLabel">
     <property name="text">
      <string>Paint stratigraphic order or cumulative thickness onto geology polygons</string>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QFormLayout" name="formLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="geologyLayerLabel">
       <property name="text">
        <string>Geology Polygon Layer:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QgsMapLayerComboBox" name="geologyLayerComboBox">
       <property name="allowEmptyLayer">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="unitNameFieldLabel">
       <property name="text">
        <string>Unit Name Field:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QgsFieldComboBox" name="unitNameFieldComboBox"/>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="stratColumnLayerLabel">
       <property name="text">
        <string>Stratigraphic Column Layer:</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QgsMapLayerComboBox" name="stratColumnLayerComboBox">
       <property name="allowEmptyLayer">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="stratUnitFieldLabel">
       <property name="text">
        <string>Strat Column Unit Field:</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QgsFieldComboBox" name="stratUnitFieldComboBox"/>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="stratThicknessFieldLabel">
       <property name="text">
        <string>Strat Column Thickness Field:</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QgsFieldComboBox" name="stratThicknessFieldComboBox">
       <property name="allowEmptyFieldName">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="5" column="0">
      <widget class="QLabel" name="paintModeLabel">
       <property name="text">
        <string>Paint Mode:</string>
       </property>
      </widget>
     </item>
     <item row="5" column="1">
      <widget class="QComboBox" name="paintModeComboBox"/>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QPushButton" name="runButton">
     <property name="text">
      <string>Run</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QgsMapLayerComboBox</class>
   <extends>QComboBox</extends>
   <header>qgis.gui</header>
  </customwidget>
  <customwidget>
   <class>QgsFieldComboBox</class>
   <extends>QComboBox</extends>
   <header>qgis.gui</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
